src/jdk.jlink/share/classes/jdk/tools/jlink/resources/plugins_zh_CN.properties

```diff
@@ -185,7 +185,7 @@ main.plugin.class=\u63D2\u4EF6\u7C7B
 
 main.plugin.module=\u63D2\u4EF6\u6A21\u5757
 
-main.plugin.category=\u7C7B\u522B
+main.plugin.category=\u7C7B\u522B 
```

Comment:
As Naoto pointed out, it looks like a trailing space was added by accident. This is also done in the ja version but for a different value. And no space was added for any value for the de version. It looks like the translation drop that Damon is receiving contains random changes.

Reply:
Additionally, there were changes made to plugins.properties in October that were not translated in the translation drop

Reply:
In our discussion, Kate mentioned that some languages are double byte, so this extra space may be correct but she hasn't looked in detail yet. I looked through all of the changes and these two instances of trailing whitespaces were the only two. I removed the extra spaces to reduce the changes as it seems useless due to the previous version not having the whitespace.